cypress/integration/visual-test/popovers-test.spec.js

```diff
@@ -0,0 +1,25 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'popovers',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+
+  it('check popovers on focus', () => {
+    cy.visit(baseURL + 'popovers');
+    cy.get('clr-dummy-anchor input:first-child').click();
```

Comment:
I haven't investigated this, but I'm seeing that the popovers don't appear, is the selector right or does the popover not show fast enough for the snapshot?

Reply:
I can see the popovers opening, are you expecting something else here?
![popover](https://user-images.githubusercontent.com/22325351/54472178-79b88580-4781-11e9-9fd7-d66e8c2e678d.png)

Reply:
You're right, I was confused!